docs/reference/deploy/config/master-config-reference.rst

```diff
@@ -96,18 +96,18 @@ values:
 
 A list of environment variables that will be set in every task container. Each element of the list
 should be a string of the form ``NAME=VALUE``. See :ref:`environment-variables` for more details.
-Environment variables specified in experiment config will override default values specified here.
-You can customize environment variables for CUDA (NVIDIA GPU), CPU, and ROCm (AMD GPU) tasks
-differently by specifying a dict with ``cuda`` (``gpu`` prior to 0.17.6), ``cpu``, and ``rocm``
-keys.
+Environment variables specified in the experiment configuration will override default values
+specified here. You can customize environment variables for CUDA (NVIDIA GPU), CPU, and ROCm (AMD
+GPU) tasks differently by specifying a dict with ``cuda`` (``gpu`` prior to Determined 0.17.6),
```

Comment:
How long before we drop references to such an old release.   That was probably useful initially, but seems more like just clutter now.

Reply:
true, we need a guideline, maybe 2 years back, i'll propose a guideline and then we'll check all docs, make it automated